File: solutions/2698-find-the-array-concatenation-value/solution.py
class Solution(object):
    def findTheArrayConcVal(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        s=0
        for i in range(len(nums)//2):
            s+=int(str(nums[i])+str(nums[-(i+1)]))
        if len(nums)%2!=0:
            s+=nums[len(nums)//2]
        return s

File: solutions/2698-find-the-array-concatenation-value/solution.java
class Solution {
    public long findTheArrayConcVal(int[] nums) {
        long s=0;
        for(int i=0;i<nums.length/2;i++)
            s+=Integer.parseInt(String.valueOf(nums[i])+String.valueOf(nums[nums.length-(i+1)]));
        if (nums.length%2!=0)
            s+=nums[nums.length/2];
        return s;
    }
}
